resources/qml/Preferences/GeneralPage.qml

```diff
@@ -377,6 +379,40 @@ UM.PreferencesPage
                 }
             }
 
+            Item
+            {
+                //: Spacer
+                height: UM.Theme.getSize("default_margin").height
+                width: UM.Theme.getSize("default_margin").width
+            }
+
+            Label
+            {
+                font.bold: true
+                text: catalog.i18nc("@label", "Override Profile")
+            }
+
+            UM.TooltipArea
+            {
+                width: childrenRect.width;
+                height: childrenRect.height;
+
+                text: catalog.i18nc("@info:tooltip", ".")
```

Comment:
Shouldn't the tooltip actually give some helpfull information? If it only displays a dot, we might as well leave it out all together.